xapian-core/cluster/cluster.cc

```diff
@@ -132,92 +141,357 @@ TermListGroup::get_termfreq(const string &tname) {
 }
 
 doccount
-ClusterSet::num_of_clusters() {
-    LOGCALL(API, doccount, "ClusterSet::num_of_clusters()", "");
+ClusterSet::size() {
```

Comment:
@ojwb Yes I've made the functions which don't touch the internal state of the object 'const'. Would be great if you could have a look into it and let me know whether I've missed out on anything :)